src/odb.h

```diff
@@ -127,6 +131,13 @@ int git_odb__read_header_or_object(
 	git_odb_object **out, size_t *len_p, git_object_t *type_p,
 	git_odb *db, const git_oid *id);
 
+/*
+ * Attempt to get the ODB's commit graph. This object is still owned by the
+ * ODB. If the repository does not contain a commit graph, it will return zero
+ * and `*out` will be set to NULL.
+ */
+int git_odb__get_commit_graph(git_commit_graph_file **out, git_odb *odb);
```

Comment:
🤔  I think that returning ENOTFOUND would be a little more idiomatic in this case.